### ISSUE TITLE
[BUG] Breaking Change in chakraui/icon

### ISSUE THREAD

**Issue by elvinvibecartons** [csandman/chakra-react-select#276]
### Description

New 3.2.0 release of Chakra UI Icon deprecates default export in favor of named exports. This breaks chakra react select which expects a default export.

### chakra-react-select Version

4.7.0

### TypeScript?

- [ ] Yes I use TypeScript

### Steps to reproduce

Run npm update. 
Chakra-React-Select will update dependency to 3.2.0 and break.

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS/iPadOS
- [ ] Android

**Comment by csandman** [comment]
Thanks for the heads up, I'll look into getting a fix out on monday!

**Comment by Naman-Garaaz** [comment]
Yes, We are facing the same issue from last night.

**Comment by atishay100** [comment]
I too, am facing the same issue

**Comment by jonashger** [comment]
@csandman Can you review #277?

**Comment by Sahil-Rally** [comment]
@csandman  Can you pls share till when this will be resolved ?

**Comment by csandman** [comment]
Sorry for the delay all, a fix is up in [`v4.7.1`](https://github.com/csandman/chakra-react-select/releases/tag/v4.7.1)! Just so everyone knows, if they run into a future like this in the future, it's easy enough to work around by overriding the package versions that CRS depends on.

For example, you can add this to your `package.json`:

```json
  "overrides": {
    "chakra-react-select": {
      "@chakra-ui/icon": "3.1.0"
    }
  }
```

You may need to wipe our your `package-lock.json`/`yarn.lock` and `node_modules`, and reinstall, but it will fix your problem until a fix can be pushed.
